src/OpenLoco/Company.cpp

```diff
@@ -1,12 +1,14 @@
 #include "Company.h"
 #include "Entities/EntityManager.h"
+#include "GameCommands/GameCommands.h"
 #include "Graphics/Gfx.h"
 #include "Interop/Interop.hpp"
 #include "Localisation/FormatArguments.hpp"
 #include "Localisation/StringIds.h"
 #include "Map/TileManager.h"
 #include "Ui/WindowManager.h"
 #include "Vehicles/Vehicle.h"
+#include <OpenLoco/GameState.h>
```

Comment:
Use relative paths for OpenLoco includes.

Reply:
fixed